Resources/Prototypes/Maps/shoukou.yml

```diff
@@ -12,7 +12,7 @@
           mapNameTemplate: '{0} Shōkō "Little Port" {1}'
           nameGenerator:
             !type:NanotrasenNameGenerator
-            prefixCreator: 'DV'
+            prefixCreator: '14'
```

Comment:
what

Reply:
Shoukou file is just trolling at this point

Reply:
I checked out to master for this file specifically. It gave me this. I manually fixed the prefix. Shoukou is no more